packages/tasit-sdk/src/helpers/DecentralandUtils.js

```diff
@@ -188,6 +197,14 @@ export default class DecentralandUtils {
       ...filter,
       fromBlock,
     });
-    return logs.map(log => iface.parseLog(log));
+
+    const parsedLogs = logs.map(log => {
```

Comment:
This function is definitely low-level enough that we'll eventually want to move it to the Tasit SDK

Reply:
Should I extract it to an issue or create a comment is enough?

Reply:
Up to you! No preference

Reply:
Added to already created issue https://github.com/tasitlabs/TasitSDK/issues/230

Reply:
Perfect. Resolving this.